tests/validation/validate_ball_and_stick.hpp

```diff
@@ -37,7 +35,7 @@ void run_ncomp_convergence_test(
         {"dt", dt},
         {"sim", "nestmc"},
         {"units", "mV"},
-        {"backend", LoweredCell::backend::name()}
+        {"backend", backend==backend_policy::use_multicore? "cpu" : "gpu"}
```

Comment:
We should either change the key to "backend_policy" (or similar) and record a representation of the policy directly, or add an interface to `model` that allows us to query the lowered cell name and use that to set the field below (line 53, or perhaps more cleanly, in a once off call before the loop, e.g. `model{singleton_recipe{c}, backend}.cell_implementation(0)`).